styleguide/source/assets/scss/03-organisms/_resource-tabs.scss

```diff
@@ -0,0 +1,189 @@
+.ama__resource-tabs {
+  height: 100%;
+}
+
+.ama__resource-tabs__nav {
+  position: relative;
+  width: 100%;
+  list-style-type: none;
+  background: $purple;
+  display: flex;
+  overflow-x: scroll;
+}
+
+//// Fake the purple tabs background extending to the right edge of the viewport.
+//.ama__resource-tabs__nav:after {
+//  display: none;
+//  @include breakpoint($bp-small) {
+//    display: block;
+//    position: absolute;
+//    background: $purple;
+//    width: 100%;
+//    top: 0;
+//    right: 0;
+//    transform: translateX(100%);
+//    height: 100%;
+//    content: "";
+//  }
+//}
+
+.ama__resource-tabs__nav li {
+  display: block;
+  padding: ($gutter / 2) ($gutter / 4);
+  color: $white;
+  text-align: center;
+  margin: 0;
+  border-right: solid 1px $white;
+
+  &.--is-active {
+    background: $white;
+    color: $purple;
+    border-right: double 3px $purple;
+    border-left: solid 1px;
+
+    + li {
+      padding-left: 4px;
+    }
+  }
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.ama__resource-tabs__nav__icon {
+  margin: 0 auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 40px;
+  height: 40px;
+}
+
+li {
+  &[data-resource='resources'] {
+    display: block;
+    @include breakpoint($bp-small) {
+      display: none;
+    }
+
+    .ama__resource-tabs__nav__icon {
+      background-image: url('../icons/svg/icon-resources-nav-resources.svg');
+    }
+  }
+
+  &[data-resource='media'] .ama__resource-tabs__nav__icon {
```

Comment:
This can be simplified by using the_ icon.scss in scss/atom/_icon.s.scss. Just add your icons in assets/icons/svg. Then add the name of the icon into _icon.scss

```
$icons: (
    alert,
    arrow,
    audio,
    checkbox,
    checkmark,
    clear,
    close,
    closedropdown,
    closedropdownsmall,
    cme,
    completed,
    download,
    edit,
    external,
    external-site,
    error,
    grid,
    image-gold,
    image-purple,
    hide,
    information,
    internal-site,
    list,
    next,
    opendropdown,
    opendrownsmall,
    pdf,
    play,
    pdf,
    podcast,
    previous,
    radio,
    search,
    select,
    show,
    usermenu,
    valid,
    video,
    viewless,
    viewmore

```

this will autogenerate the icon background for you. 

To include the icon you either use the class .icon--resources-nav-resources or extend it.